frontend/src/store/plugins/mediaPlugin.js

```diff
@@ -6,17 +6,20 @@
 
 export default function (vuetify) {
   return store => {
+    const mq = window.matchMedia('(prefers-color-scheme: dark)')
+    // set the preferred theme according to user settings and watch for changes
     const setPrefersColorScheme = dark => {
       store.commit('SET_PREFERS_COLOR_SCHEME', dark ? 'dark' : 'light')
     }
-    const mq = window.matchMedia('(prefers-color-scheme: dark)')
-    mq.addEventListener('change', e => setPrefersColorScheme(e.matches))
     setPrefersColorScheme(mq.matches)
-
-    const colorScheme = (state, getters) => getters.colorScheme
+    mq.addEventListener('change', e => setPrefersColorScheme(e.matches))
+   
+    // apply the theme initially and watch for changes
     const setDarkTheme = value => {
       vuetify.framework.theme.dark = value === 'dark'
     }
+    const colorScheme = (state, getters) => getters.colorScheme
+    setDarkTheme(store.getters.colorScheme)
```

Comment:
This was missing in the original code and that was the reason why it has not worked. The other changes do not change anything. I do not like code comments, if the code is self explaining https://medium.com/@bpnorlander/stop-writing-code-comments-28fef5272752.

Reply:
> There are only four situations I can think of at the moment where I need to comment code:
> 
> 1. In the styles of Javadoc, RubyDoc, et cetera for documenting APIs others will use.
> 2. In the off chance it really is that complex: For example, on a bioinformatics DNA search function that took 5 weeks to formulate and write out. That’s how rare it is to have something complex enough to warrant comments.
> 3. TODOs, which should be the exception, not the rule
> 4. Explaining why the most obvious code wasn’t written. (Design decisions)
> 

[Sam Larbi](http://www.codeodor.com/index.cfm/2008/6/18/Common-Excuses-Used-To-Comment-Code-and-What-To-Do-About-Them/2293)